### ISSUE TITLE
`--createstub` incorrectly copies optional trailing comma in bracketed parameter lists

### ISSUE THREAD

**Issue by mniip** [microsoft/pyright#3405]
**Describe the bug**
In type-level parameter lists such as `Literal[1, 2]` an optional trailing comma is allowed as in `Literal[1, 2,]`. Yet when a type like this is encountered in a file processed by `--createstub`, the produced stub file contains the comma in an incorrect place (outside the brackets as opposed to inside), yielding syntactically invalid python.

**To Reproduce**
Run `pyright --createstub a` on the `a.py` file from the Code section. Observe that the resulting `pyi` file is not syntactically valid.

**Expected behavior**
The `pyi` file should be valid python, with, or without the trailing comma.

**Screenshots or Code**
```py
# a.py
from typing import Literal
x: Literal[1, 2,]
```
```py
# typings/a/__init__.pyi
"""
This type stub file was generated by pyright.
"""

from typing import Literal

x: Literal[1, 2],
```

**VS Code extension or command-line**
Commandline pyright 1.1.241

**Additional context**

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.242, which I just published. It will also be included in the next release of pylance.
